ingest_file.py

```diff
@@ -22,11 +30,11 @@ def ingest_files(queue, refdes, method, deployment, filenames):
 
 
 if __name__ == '__main__':
-
-    queue = sys.argv[1]
-    refdes = sys.argv[2]
-    method = sys.argv[3]
-    deployment = int(sys.argv[4])
-    filenames = sys.argv[5:]
+    # Get the command line options
+    queue = options['<queue>']
+    refdes = options['<refdes>']
+    method = options['<method>']
+    deployment = int(options['<deployment>'])
+    filenames = [options['<filenames>']]
```

Comment:
Will this only match the first filename, or will it match all remaining arguments on the command line?